src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/tableView.js

```diff
@@ -0,0 +1,249 @@
+class TableViewVisualization extends Visualization {
+    static inputType = "Any"
+
+    onDataReceived(data) {
+
+        var tableOf = function (content, level) {
+            var open = '<table class="level' + level + '">';
+            return open + content + "</table>";
+        }
+
+        var hasExactlyKeys = function (keys, obj) {
+            return Object.keys(obj).length == keys.length && keys.every(k => obj.hasOwnProperty(k));
+        };
+
+        var getAtNestedKey = function (data, key) {
+            var res = data;
+            key.forEach(function (k) { res = res[k]; });
+            return res;
+        }
+
+        var repNestedKey = function (key) {
+            return key.join(".");
+        }
+
+        var generateNestings = function (data, key) {
+            var first = getAtNestedKey(data[0], key);
+            if (!(first instanceof Object)) return [key];
+            var firstKeys = Object.keys(first);
+            var isNestable = data.every(obj => hasExactlyKeys(firstKeys, getAtNestedKey(obj, key)));
```

Comment:
not aligned `=` - here and in other places

Reply:
:o